tests/unit/datepicker/core.js

```diff
@@ -540,4 +540,83 @@ QUnit.test( "mouse", function( assert ) {
 		"Mouse click inline - next" );
 } );
 
+QUnit.test( "position", function( assert ) {
+	assert.expect( 4 );
+
+	var $container = $( "#qunit-fixture" ),
+		$modal = $container.children('div'),
+		$inputOne = testHelper.init( "#inp" ),
+		$inputTwo = testHelper.init( "#alt" );
+
+	// Undo the weird positioning on the container
+	$container.css({
+		position: "static",
+		top: 0,
+		left: 0,
+		height: "5000px",
+	});
+
+	// Position the modal in the middle of the screen
+	$modal.css({
+		display: "none", // Initially must be hidden
+		position: "fixed",
+		"overflow": "hidden auto",
+		top: "50%",
+		left: "50%",
+		right: "auto",
+		width: "500px",
+		height: "500px",
+		marginTop: "-100px",
+		marginLeft: "-100px"
+	});
+
+	// Add an internal wrapper around all the children nodes
+	$modal.wrapInner(`<div id="relative-wrapper" style="position: relative"></div>`)
+	var $wrapper = $modal.children('#relative-wrapper');
+	$wrapper.prepend(`<div style="height: 300px;"></div>`);
+
+	// Wrap inputs with a container
+	$inputOne.wrap(`<div id="input-one-container" style="height: 200px;"></div>`);
+	$inputTwo.wrap(`<div id="input-two-container" style="height: 200px;"></div>`);
+
+	assert.ok( $modal.is(":hidden"), "Modal is hidden" );
+
+	// Set the page scroll position way down the page
+	var done = assert.async();
```

Comment:
If the test is async, please put this declaration at the top of the test, as in other tests.